tests/zfs-tests/include/properties.shlib

```diff
@@ -91,8 +91,13 @@ function get_rand_large_recsize
 #
 # Functions to toggle on/off properties
 #
-typeset -a binary_props=('atime' 'devices' 'exec' 'nbmand' 'readonly' 'setuid'
-    'xattr' 'zoned')
+if [[ $(linux_version) -ge $(linux_version "4.4") ]]; then
+	typeset -a binary_props=('atime' 'devices' 'exec' 'readonly' 'setuid'
```

Comment:
You can selectively add 'nbmand' to the array rather than defining it for both cases:
```
# Only older kernels support non-blocking mandatory locks
if [ $(linux_version) -lt $(linux_version "4.4") ] ; then
        binary_props+=('nbmand')
fi
```